test/playwright/test_ux.py

```diff
@@ -118,7 +118,7 @@ def _create_new_environment(page, screenshot=False):
     
     # Interact with the environment shortly after creation
     # click to open the Active environment dropdown manu
-    page.get_by_role("button", name=" - Active", exact=False).click()
+    page.get_by_text(" - Active", exact=False).click()
```

Comment:
This is necessary because the generated `Select` component is now a `combobox`, so the `button` role doesn't match.

Using `get_by_text` might look error-prone here, but I think it's okay. The `option` line below will error out if `get_by_text` matches something else here.

Reply:
Note: I use `get_by_text` and not `get_by_role("combobox", ...)` here because playwright fails to find this button with the latter approach.